models/staging/jaffle_shop/stg_orders.sql

```diff
@@ -5,8 +5,9 @@ with orders as (
         user_id as customer_id,
         order_date,
         status
-
+        --analytics.analytics.orders amount

```

Comment:
I recommend removing this line to avoid any confusion

Reply:
this is in relation to the other issue where I was trying to get amounts to be part of this model so that I can use the ref orders in dim_customers. ill look into this as well. thanks

Reply:
ah I should be doing that fct_orders level not staging in which case I can use fct_orders model instead of stg_model in my orders CTE in dim_customers. will try that